catalog/model.go

```diff
@@ -37,6 +37,7 @@ type CommitLog struct {
 	Message      string    `db:"message"`
 	CreationDate time.Time `db:"creation_date"`
 	Metadata     Metadata  `db:"metadata"`
+	MetaRangeID  string    `db:"meta_range_id"`
```

Comment:
DDL upgrade/downgrade needed?

Reply:
No, meta_range_id column introduced in [DDL 20](https://github.com/treeverse/lakeFS/blob/master/ddl/000020_tree_to_range.up.sql#L3).